src/ansys/additive/core/thermal_history.py

```diff
@@ -37,80 +38,91 @@
 
 
 class Range:
-    """Defines a parameter that spans a range of values.
+    """Defines a range of values."""
 
-    min: float
-        Minimum value.
-    max: float
-        Maximum value.
-    """
-
-    def __init__(self, **kwargs):
+    def __init__(self, min: float, max: float):
         """Initialize a ``Range`` object."""
-        self.min = 0
-        self.max = 0
-        for key, value in kwargs.items():
-            getattr(self, key)  # raises AttributeError if key not found
-            setattr(self, key, value)
+        self._min = min
+        self._max = max
         if self.min > self.max:
             raise ValueError("Attempted to initialize Range with min greater than max")
 
     def __repr__(self):
         repr = type(self).__name__ + "\n"
         for k in self.__dict__:
-            repr += k + ": " + str(getattr(self, k)) + "\n"
+            repr += k.replace("_", "", 1) + ": " + str(getattr(self, k)) + "\n"
         return repr
 
     def __eq__(self, other: object) -> bool:
         if not isinstance(other, Range):
             return False
-        for k in self.__dict__:
-            if getattr(self, k) != getattr(other, k):
-                return False
-        return True
+        return self._min == other._min and self._max == other._max
+
+    @property
+    def min(self) -> float:
+        """Minimum value of range."""
+        return self._min
+
+    @property
+    def max(self) -> float:
+        """Maximum value of range."""
+        return self._max
 
     def _to_range_message(self) -> RangeMessage:
-        """Create a ``RangeMessage`` to send to the server based upon this
-        object."""
+        """Transform this object into a RangeMessage to send to the server."""
         return RangeMessage(min=self.min, max=self.max)
 
 
 class CoaxialAverageSensorInputs:
-    """Provides descriptions for coaxial average sensors.
+    """Provides descriptions for coaxial average sensors."""
 
-    radius: float
-        Radius in meters for the circular field of the view of sensor. Valid values
-        are from 5e-5 to 1.5e-2 m (0.05 - 15 mm).
-    z_heights: Range[]
-        Array of ranges in meters along the z axis of the geometry. The simulated
-        sensor follows the scan path for each deposit layer within each range.
-    """
+    MIN_SENSOR_RADIUS = 5e-5
+    """Minimum radius for the circular field of view of sensor (m)."""
```

Comment:
```suggestion
    """Minimum radius for the circular field of view of the sensor (m)."""
```
Here, and in the next few instances, are examples where adding the article really improves readability!